frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewSidePanel/FeatureOverviewSidePanelDetails/FeatureOverviewSidePanelDetails.tsx

```diff
@@ -0,0 +1,49 @@
+import { IFeatureToggle } from 'interfaces/featureToggle';
+import { styled } from '@mui/material';
+import { useLocationSettings } from 'hooks/useLocationSettings';
+import { formatDateYMD } from 'utils/formatDate';
+import { parseISO } from 'date-fns';
+// import { UserAvatar } from 'component/common/UserAvatar/UserAvatar';
```

Comment:
Can we yeet the commented out code?

Reply:
Yep, was something we were not sure we should include, but it seems we currently don't have this information on the back-end, so we'll leave it out for now. Addressed in https://github.com/Unleash/unleash/pull/2489/commits/71e0ade5beb50aa38b2972d942e26f0102eee8f0